samples/CallingStateful/src/components/CallManager.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Call, TeamsCall } from '@azure/communication-calling';
+import { IStackStyles, PrimaryButton, Stack, Text, Theme, useTheme } from '@fluentui/react';
+import React, { useEffect, useState } from 'react';
+
+export interface CallManagerProps {
+  onSetResume: (call: Call | TeamsCall) => void;
+  calls: Call[] | TeamsCall[];
+  activeCall?: Call | TeamsCall;
+  onSetHold: (call: Call | TeamsCall) => void;
+  onEndCall: (call: Call | TeamsCall) => void;
+}
+
+export const CallManager = (props: CallManagerProps): JSX.Element => {
+  const { onSetResume, calls, onSetHold, activeCall } = props;
+  const theme = useTheme();
+  return (
+    <Stack verticalAlign={'start'} styles={callManagerContainerStyle(theme)}>
+      <Text variant="xLarge">Calls</Text>
+      <Stack>
+        {activeCall && (
+          <Stack>
+            <Text variant="large">Active Call</Text>
+            <CallItem call={activeCall} onSetResume={onSetResume} isActiveCall={true} onSetHold={onSetHold} />
+          </Stack>
+        )}
+        {
+          <Stack>
+            {calls && calls.length > 1 && <Text variant="large">Other Calls</Text>}
+            {calls.map((call) => {
+              if (call.id === activeCall?.id) {
+                return;
+              }
+              return (
+                <CallItem
+                  key={call.id}
+                  call={call}
+                  onSetResume={onSetResume}
+                  isActiveCall={call.id === activeCall?.id}
+                  onSetHold={onSetHold}
+                />
+              );
+            })}
+          </Stack>
+        }
+      </Stack>
+    </Stack>
+  );
+};
+
+interface CallItemProps {
+  call: Call | TeamsCall;
+  isActiveCall: boolean;
+  onSetResume: (call: Call | TeamsCall) => void;
+  onSetHold: (call: Call | TeamsCall) => void;
+}
+
+const CallItem = (props: CallItemProps): JSX.Element => {
+  const { call, onSetResume, onSetHold } = props;
+
+  const [callTitle, setCallTitle] = useState<string>('');
+
+  useEffect(() => {
+    if (call.state === 'Connecting') {
+      setCallTitle('Connecting');
+    } else {
+      const names = call.remoteParticipants.map((participant) => participant.displayName);
+      if (names.length === 1 && names[0]) {
+        setCallTitle(names[0]);
+      } else if (names.length > 1) {
+        setCallTitle(names.join(', '));
+      } else if (names.length === 0) {
```

Comment:
nit: just `else` here to ensure we don't leave the string in the Connecting state
```suggestion
      } else {
```